tests/e2e/test_tools.py

```
@@ -72,10 +94,7 @@ async def patient_id(mcp_server) -> str | None:
         "payload": {
             "resourceType": "Patient",
             "gender": "male",
-            "name": {
-                "family": f"TestFamily-{suffix}",
-                "given": [f"TestGiven-{suffix}"],
-            },
+            "name": [{"prefix": ["Mr."], "family": f"TestFamily-{suffix}", "given": ["TestGiven"]}],
         },
     }
     logger.debug("[TOOL REQUEST] create:", request_payload)
```

Ruff (0.15.12)

[error] Too many arguments for `logging` format string

(PLE1205)